package.json

```diff
@@ -144,7 +144,7 @@
     "eslint-plugin-react": "7.28.0",
     "eslint-plugin-react-hooks": "4.3.0",
     "good-fences": "1.0.1",
-    "husky": "7.0.4",
+    "husky": "^7.0.4",
```

Comment:
Remove the ^ from this package please. We pin the version, so we are guaranteed the same version on each install